tests/e2e/asyncOps.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {test} from '@playwright/test'
+import { FormElements } from '../page-objects/formElements'
+import { asyncOpsElements } from '../page-objects/asyncOPS'
```

Comment:
Don't abbreviate operations if you don't have to. No real reason to shorten text.